src/TitleReceive/TitleReceiveList.js

```diff
@@ -30,9 +31,13 @@ const visibleColumns = [
   'isCreateItem',
 ];
 
+const columnWidths = {
+  location: '250px',
```

Comment:
why 250 ?

Reply:
for some reasons MCL autosize doesn't work properly for Select field